### ISSUE TITLE
Don't use apt-key

### ISSUE THREAD

**Issue by theFeu** [Icinga/get-started-docs#2]
https://github.com/Icinga/get-started-docs/blob/a1b88fa52d9c091143dcc9f6c5bcd6c5123773e5/doc/include/tab-repository.md?plain=1#L7

@[Al2Klimov](https://github.com/Al2Klimov) [on May 15](https://github.com/Icinga/get-started-docs/pull/1#discussion_r1601641684)
This is insecure, see https://blog.cloudflare.com/dont-use-apt-key

**Comment by jrauh01** [comment]
@Al2Klimov would following be a better option?
`wget -O - https://packages.icinga.com/icinga.key | gpg --dearmor -o /usr/share/keyrings/icinga-archive-keyring.gpg`

**Comment by RincewindsHat** [comment]
If Icinga does not yet provide a keyring archive, please do not save the key to `/usr/share/...` since this is intented to be managed automatically not manually.
`/etc/apt/keyrings` might be the one you want.

**Comment by Al2Klimov** [comment]
No, once we have a keyring, it shall take over the control over the file.
